README.md

```diff
@@ -1,5 +1,6 @@
 # Objectivity.AutoFixture.XUnit2.AutoMoq
 [![Build Status](https://travis-ci.org/ObjectivityLtd/AutoFixture.XUnit2.AutoMoq.svg?branch=master)](https://travis-ci.org/ObjectivityLtd/AutoFixture.XUnit2.AutoMoq)
+[![Downloads](http://nuget-stats-api.jankowskimichal.pl/api/badges/Objectivity.AutoFixture.XUnit2.AutoMoq/totalDownloads)](https://github.com/ObjectivityLtd/AutoFixture.XUnit2.AutoMoq)
```

Comment:
Why do we host the nuget api on jankowskimichal.pl domain and why is it unsecured connection? Consider using https://shields.io/